docs/source/index.rst

```diff
@@ -1,7 +1,47 @@
-Welcome to the libMobility Documentation!
-=========================================
+libMobility: GPU Solvers for Hydrodynamic Mobility
+==================================================
 
+libMobility is a C++ library with Python bindings offering several GPU solvers that can compute the action of the hydrodynamic mobility (at the RPY/FCM level) of a group of particles (in different geometries) with forces and torques acting on them.
 
+Functionality
+-------------
+
+Given a group of forces, :math:`\boldsymbol{F}`, and torques, :math:`\boldsymbol{\tau}`, acting on a group of positions, :math:`\boldsymbol{X}` and directions :math:`\boldsymbol{\tau}`, the libMobility solvers can compute:
+
+.. math::
+
+   \begin{bmatrix}\boldsymbol{dX}\\\boldsymbol{d\tau}\end{bmatrix} = \boldsymbol{\mathcal{\Omega}}\begin{bmatrix}\boldsymbol{F}\\\boldsymbol{T}\end{bmatrix}dt + \text{prefactor}\sqrt{2 k_B T \boldsymbol{\mathcal{\Omega}}}d\boldsymbol{W}
+
+Where:
+
+- :math:`d\boldsymbol{X}` are the linear displacements
+- :math:`\boldsymbol{d\tau}` are the angular displacements
+- :math:`\boldsymbol{\mathcal{\Omega}}` is the grand mobility tensor
```

Comment:
I think it may be clearer to stick to \mathcal{M} instead of \mathcal{\Omega} for the mobility matrix due to how we name our functions (e.g. Mdot, SqrtMdotW)

Reply:
You are right. Lets keep M.